### ISSUE TITLE
Tails boot failure: "no supported filesystem images found at /live"

### ISSUE THREAD

**Issue by EstevaoCostaG3** [ventoy/Ventoy#43]
Hi there! I've tested Ventoy on a 64 bits legacy (BIOS) machine and it failed to boot some distros. Only `Windows 10`, `Porteus-XFCE-v5.0rc1-x86_64` and `rescatux-0.72-beta7` that worked.
In this issue, I'm reporting the error that I got when booting `tails-amd64-4.0`. 
This is the error that I got:
![ventoy-tails](https://user-images.githubusercontent.com/22309462/80870389-226ec080-8c74-11ea-9aba-9795bae0cb5a.jpg)

**Comment by ventoy** [comment]
Please just `cat  /ventoy/log`   and  send me the log, thanks!

**Comment by EstevaoCostaG3** [comment]
Here it's the result of `cat /ventoy/log`
![WhatsApp Image 2020-05-03 at 1 43 32 PM](https://user-images.githubusercontent.com/22309462/80921470-97a9c680-8d44-11ea-8d1f-901043768f4b.jpeg)

**Comment by ventoy** [comment]
I tested tails-amd64-4.0.iso with ventoy-1.0.08 in my laptop (legacy bios)  but it worked well :(

**Comment by EstevaoCostaG3** [comment]
This could be a hardware specific issue, maybe, I don't know. 
I'm downloading tails 4.6, which is the latest release, if it works, then it's fine for me.

**Comment by EstevaoCostaG3** [comment]
Just to let you know, @ventoy. I downloaded the Tails 4.6 image for USB, since that's what I'm using. It's a `.img` file, not `.iso`. So I copied it to my USB stick and it didn't appeared on Ventoy's menu. 
So I appended `.iso` to the file name, and it appears now, but I can't boot in both UEFI and BIOS mode with my laptop that supports those boot modes.
- In UEFI mode, I got this error after choosing tails on Ventoy's menu:
![photo5023833700537837818](https://user-images.githubusercontent.com/22309462/81483228-90822d00-920a-11ea-9a02-c928db307439.jpg)
- In BIOS mode, this is the error:
![photo5023833700537837817](https://user-images.githubusercontent.com/22309462/81483237-9f68df80-920a-11ea-911f-0d6911e770a7.jpg)
So, a possible workaround it would be to download the version of `tails-amd64-4.6` for DVDs, since it's a `.iso` file. That's what I'm going to try next.

**Comment by EstevaoCostaG3** [comment]
> So, a possible workaround it would be to download the version of `tails-amd64-4.6` for DVDs, since it's a `.iso` file. That's what I'm going to try next.

I've tested it, and it didn't work.
- In BIOS mode I got the following error:
![bios mode error](https://user-images.githubusercontent.com/22309462/81484411-36d23080-9213-11ea-9681-5127fe4beb45.jpeg)
- In UEFI mode I couldn't find the `/ventoy/log` file (as you can see in the figure below, there's no `/ventoy` diretory):
![index](https://user-images.githubusercontent.com/22309462/81484428-5a957680-9213-11ea-9ae6-a11a54c1b2ec.jpeg)

Let me know If you need some specs about the laptop that I used this time (and in the previous comment as well).

**Comment by Perkolator** [comment]
I just tried to boot `tails-amd64-4.8.iso` in UEFI mode and it says: "no bootfile found for UEFI". Secure boot disabled in bios and ventoy.

Everything else I tried has been working just fine. Ventoy is the first tool that actually has worked for me, well, almost 100%.  :) Amazing tool! Thank you!

These other isos I tried: clonezilla live (both variants), linux mint 20 cinnamon and xfce, kde neon, systemrescuecd, win8.1.1, win10_2004, win10_LTSC_2019_eval.

Just tails wont work. Any ideas why? And should it even? I tried to google about this but I failed to gain more knowledge.

**Comment by Perkolator** [comment]
Hooray. Just updated Ventoy to 1.0.16 and tried to boot `tails-amd64-4.8.iso` and it worked!

**Comment by shigutso** [comment]
I'm having the same issue.
I am able to boot Ubuntu, UBCD, Windows 10, Android x86, etc but not Tails OS.
Very old laptop (BIOS only, no UEFI)
Ventoy 1.0.17
Tested tails-amd64-4.8.iso and tails-amd64-4.9.iso

Logs:

```
============== VENTOY =================
377 blocks
5724 blocks
kenel version=Linux version 5.7.0-1-amd64 (debian-kernel@lists.debian.org) (gcc version 9.3.0 (Debian 9.3.0-14), GNU ld (GNU Binutils for Debian) 2.34) #1 SMP Debian 5.7.6-1 (2020-06-24)
kenel cmdline=BOOT_IMAGE=/live/vmlinuz initrd=/live/initrd.img boot=live config live-media=removable nopersistence noprompt timezone=Etc/UTC block.events_dfl_poll_msecs=1000 splash noautologin module=Tails slab_nomerge slub_debug=FZP mce=0 vsyscall=none page_poison=1 init_on_alloc=1 init_on_free=1 mds=full,nosmt  noapic noapm nodma nomce nolapic nomodeset nosmp vga=normal
remove whole sbin directory
=====ventoy_unpack_initramfs: #/initrd001 0 3037 /initrd001_tmp#
vtx=3037 cat
62 blocks
=====ventoy_unpack_initramfs: #/initrd001 436 3037 /initrd001_tmp#
vtx=3037 cat
5868 blocks
=====ventoy_unpack_initramfs: #/initrd001 212 FD37 /initrd001_tmp#
vtx=FD37 xzcat
311981 blocks
Now hand over to ventoy.sh
#### install vtoytool #####
try /ventoy/tool/vtoytool/00/ ...
vtoytool_64 OK
use vtoy_fuse_iso_64
use unsquashfs_64
use unsquashfs_64
kernel version
Linux version 5.7.0-1-amd64 (debian-kernel@lists.debian.org) (gcc version 9.3.0 (Debian 9.3.0-14), GNU ld (GNU Binutils for Debian) 2.34) #1 SMP Debian 5.7.6-1 (2020-06-24)
OS=###debian###
##### distribution = tails ######
ventoy_os_install_dmsetup /dev/sdb ...
device mapper module is NOT loaded, now load it...
KO=/lib/modules/5.7.0-1-amd64/kernel/drivers/md/dm-mod.ko
dmsetup install finish, now check it...
No devices found
dmsetup work ok
create_ventoy_device_mapper /dev/sdb --readonly
dmsetup avaliable in system /sbin/dmsetup
device-mapper module check success
==== create ventoy device mapper success ====
replace block device link /dev/sdb2...
sdb2 is USB device
```

**Comment by Perkolator** [comment]
`tails-amd64-4.9.iso` working here in UEFI and GPT just fine with Ventoy 1.0.17.

**Comment by shigutso** [comment]
Just tested Ventoy 1.0.18 on the same laptop after checking the changelog
`tails-amd64-4.9.iso` now boots!

Thanks @ventoy :D

**Comment by Perkolator** [comment]
`tails-amd64-4.9.iso` working in UEFI and GPT (secure boot disabled) with Ventoy 1.0.18, BUT, it's really slow now, booting and opening programs after boot to the desktop. With Ventoy 1.0.17 it was fast/normal. Every other iso is working just fine and not slow.

**Comment by Zeik0s** [comment]
tails-amd64-4.12.iso doesn't work with Ventoy 1.0.27 🤔

**Comment by Perkolator** [comment]
> tails-amd64-4.12.iso doesn't work with Ventoy 1.0.27

It works but it is super super super slow to start and use.. like I described in my previous message. Something changed in Ventoy 1.0.18 which made Tails awfully slow.

**Comment by gthubs** [comment]
> 
> 
> I have not checked recent versions, but Tails may try to modify the USB drive on first boot? https://rmprepusb.blogspot.com/2020/01/direct-uefi64-boot-from-tails-24-using.html
No, I checked  isolinux\live64.cfg in tails-amd64-4.23.iso, there is no **union=aufs** parameter

My problem partially decided. I downloaded Tails-AMD64-4.23.img instead of Tails-AMD64-4.23.ISO, now loading in UEFI mode occurs without errors and quickly (surprisingly but USB 2 poorly affects the start time), but only when the flash drive is connected to the USB port 2 . When using USB 3.0, the problem remained, but I think the reason is not in Ventoy, but in the Tails itself or in my old computer

**Comment by ventoy** [comment]
@gthubs 
Use this CI release to see whether still with the long delay in UEFI mode for the ISO file.
https://github.com/ventoy/Ventoy/actions/runs/1359055022

**Comment by ventoy** [comment]
OK. 
Ventoy can work OK even USB hardware protect is enabled, this is an important and valuable feature of Ventoy.
So as you said, the problem may be in the Tails distribution itself since other distros work OK.
